mgmt/rest/v1/tribe.go

```diff
@@ -42,12 +42,16 @@ var (
 	ErrMemberNotFound        = errors.New("Member not found")
 )
 
+// Deprecated: Update to apiV2(https://github.com/intelsdi-x/snap-client-go).
```

Comment:
@candysmurf I previously added the notification to REST_API.md and mentioned in the issue it is pending the deprecation announcement; any other requests?